utils/networks.ts

```diff
@@ -2,23 +2,29 @@ import { isHexString } from "@ethersproject/bytes";
 
 export type NetworkConfig = {
   blockExplorerUrl: string;
+  tokenListUrl: string;
 };
 
 export const networks = {
   1: {
     blockExplorerUrl: "https://etherscan.io/",
+    tokenListUrl: "https://gateway.ipfs.io/ipns/tokens.uniswap.org",
```

Comment:
would any part of us rather store these statically?

Reply:
Good question. AFAIK we don't have a spec on what tokens specifically to support, so I'd say we can leave it as is for now and revisit/simplify in a future PR

Reply:
#43